lib/ExpressRedisCache/get.js

```diff
@@ -55,13 +55,13 @@ module.exports = (function () {
       /** If has wildcard */
       if ( hasWildcard ) {
         /** Get a list of keys using the wildcard */
-        self.client.keys(prefix + ':' + name, domain.intercept(function (keys) {
-          if ( ! keys.length ) {
+        self.client.scan(0, 'match', prefix + ':' + name, domain.intercept(function (result) {
```

Comment:
You are going to perform only one SCAN iteration which could return empty results set, so to find all matching keys you will need to repeat scanning till the end.